yt/yql/agent/config.h

```diff
@@ -129,6 +129,8 @@ class TYqlPluginConfig
 
     NYTree::INodePtr OperationAttributes;
 
+    NYTree::INodePtr Libraries;
```

Comment:
Let's make it `THashMap<TString, TString>` because this field is strongly typed.